crates/api_crud/src/user/create.rs

```diff
@@ -144,13 +150,24 @@ pub async fn register(
     .accepted_application(accepted_application)
     .default_listing_type(Some(local_site.default_post_listing_type))
     .post_listing_mode(Some(local_site.default_post_listing_mode))
-    .interface_language(language_tag)
+    .interface_language(language_tags.first().cloned())
     // If its the initial site setup, they are an admin
     .admin(Some(!local_site.site_setup))
     .build();
 
   let inserted_local_user = LocalUser::create(&mut context.pool(), &local_user_form).await?;
 
+  let all_languages = Language::read_all(&mut context.pool()).await?;
```

Comment:
Some tests are failing, which tells me something is wrong with moving this logic from the `LocalUser::create` (which is run in the unit tests), and this, which is only run from the API.